frontend/src/tests/components/Nav/AppNavbar.test.js

```diff
@@ -84,6 +84,23 @@ describe("AppNavbar tests", () => {
         expect(adminMenu).toBeInTheDocument();        
     });
 
+    test("renders correctly for admin(using admin for now until rider is implemented) user", async () => {
```

Comment:
The RIDER role is already  mostly implemented.  But if it isn't, you should do that first.  Otherwise, this page is pointless, and it shouldn't be merged into main.

There is already code in the UsersController that toggles the role:

* https://github.com/ucsb-cs156-m23/proj-gauchoride-m23-9am-2/blob/255f674f63bc6a33a54d5792802404052dcc620d/src/main/java/edu/ucsb/cs156/gauchoride/controllers/UsersController.java#L98 

You might need to add some code next to this to make sure that the role propagates: 

* https://github.com/ucsb-cs156-m23/proj-gauchoride-m23-9am-2/blob/255f674f63bc6a33a54d5792802404052dcc620d/src/main/java/edu/ucsb/cs156/gauchoride/config/SecurityConfig.java#L112

Reply:
And some code here as well: 

https://github.com/ucsb-cs156-m23/proj-gauchoride-m23-9am-2/blob/255f674f63bc6a33a54d5792802404052dcc620d/src/main/java/edu/ucsb/cs156/gauchoride/config/SecurityConfig.java#L89

Reply:
For the time being, that's probably enough to get the rider role fully established, although there's some frontend work that needs to be done as well in UsersTable, and AdminUsersPage so that the toggles work correctly.